src/store/BarrierStore.ts

```diff
@@ -83,12 +84,12 @@ export default class BarrierStore {
             hideOffscreenLine: observable,
             hideOffscreenBarrier: observable,
             isSingleBarrier: observable,
-            pip: computed,
-            yAxisWidth: computed,
-            priceLabelWidth: computed,
+            destructor: action.bound
```

Comment:
` destructor: action.bound,`
 Comma :) Or it won't build

Reply:
Thanks, yes I sorted the lines, but it didn't have comma on one of it)